profile/README.md

```diff
@@ -0,0 +1,63 @@
+# Lombiq Technologies 
+
+
+[Lombiq Technologies](https://lombiq.com) is a software, training, and services company focusing on web development with open Microsoft technologies. 
+Our main area of expertise is the Orchard Core CMS. We provide custom development, training, hosting, and consulting services and we are behind the first (and only) Orchard CMS SaaS.
```

Comment:
I'd replace "Orchard Core CMS" to "Orchard Core content management system" because "CMS" is not part of the Orchard Core branding, but it is part of the Orchard CMS branding. Also I think it's helpful to quickly resolve the acronym for people who just stumble in and not sure what's the context (or for people who came here from the Hastlayer side so they wouldn't necessarily know about web development terms).